Tests/IntegrationTests/Infrastructure/JarInfo.cs

```diff
@@ -0,0 +1,134 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="JarInfo.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using SonarQube.Common;
+using SonarQube.Plugins.Common;
+using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace SonarQube.Plugins.IntegrationTests
+{
+    [XmlRoot("JarInfo")]
+    public class JarInfo
```

Comment:
I'll add a comment. "Data classes to load the XML file created by the PluginInspector"